### ISSUE TITLE
A lot of errors, latest Windows 11 24H2 

### ISSUE THREAD

**Issue by capsload2** [ntdevlabs/tiny11builder#310]
Im getting a lot of errors, trying to build a Tiny11 with the latest Windows 11 Pro.

![err](https://github.com/user-attachments/assets/c6a5772e-a3fd-4b9f-9a0a-d646a42f7b31)

**Comment by Vanqy9316** [comment]
thanks for submitting this issue. can you please verify that you have started the script as an administrator? 
Even if you end up getting a .iso file at the end of the script, it will not work unless its on full privileges.

Please try again with the following steps:
- copy the script file, e.g. `C:\Users\Vanqy\Desktop\Tools\tiny11builder-main` to clipboard
- Start PowerShell/Terminal as admin via the start menu or powertoys run. 
- type `powershell -f` ,paste from clipboard and hit enter. the full command should look like this `PowerShell -f C:\Users\Vanqy\Desktop\Tools\tiny11builder-main`

when the script is done, notice the size difference of the resulting .iso

the last two days, ive tested multiple times of not using and using `powershell -f` to run the script. without the command, even in a admin privileged powershell, the resulting .iso was not working and failed in midst of setup: "cannot validate product key".
upon search on the internet, i stumbled upon this: [system can't access the file on the usb flash drive to validate the key](https://answers.microsoft.com/en-us/windows/forum/all/windows-11-pro-new-pc-couldnt-verify-the-product/0f3eee7b-06b1-4b07-90fa-09ea8237c079)

cheers

**Comment by robinhood2014** [comment]
> Please try again with the following steps:
> 
> * copy the script file, e.g. `C:\Users\Vanqy\Desktop\Tools\tiny11builder-main` to clipboard
> * Start PowerShell/Terminal as admin via the start menu or powertoys run.
> * type `powershell -f` ,paste from clipboard and hit enter. the full command should look like this `powershell -f C:\Users\Vanqy\Desktop\Tools\tiny11builder-main`

That got it working. It would help the user greatly if we were to explicitly specify that this tool requires the built-in Windows PowerShell, not the new cross-platform PowerShell Core. Also, we don't want to forget the name of the actual .ps1 file in the path, thus: `powershell -f C:\path\to\tiny11builder-main\tiny11maker.ps1` or `powershell -f C:\path\to\tiny11builder-main\tiny11Coremaker.ps1`.

**Comment by Karl-WE** [comment]
@robinhood2014 @Vanqy9316 
the tiny11maker (PowerShell) the script works with PS 5.1 and PS 7.x in PowerShell. Last check PS 7.4.
Please elaborate.

**Comment by Vanqy9316** [comment]
@Karl-WE i do not know if i speak for OP or anyone else, but i elaborated the issue and my solution as good as i could. If you require more specific information which i am unaware of, please disclose, so further steps can be taken to solve this issue fully.

**Comment by Vanqy9316** [comment]
@Karl-WE 

unknown command:
"gci"

**Comment by Karl-WE** [comment]
Are you sure you are executing the script in Windows 10/11 Microsoft Terminal?
Error indicates you are in a legacy command line (cmd).

**Comment by Vanqy9316** [comment]
@Karl-WE 
yes, i am using the new Microsoft Terminal. The error is not related to PowerShell afaik. See attached picture: 

![Image](https://github.com/user-attachments/assets/8f84bf6b-dd23-49cf-9f98-c127753d403d)

**Comment by Karl-WE** [comment]
OK I see the point. Please go to Windows Terminal Settings and change default to PowerShell.

This will do it. Appears like you are on Windows 10 where cmdline was still the default.

**Comment by Karl-WE** [comment]
OK I see the point. Please go to Windows Terminal Settings and change default to PowerShell.

This will do it.

**Comment by Karl-WE** [comment]
Oh dear. I think this is out of my hands.
We need help and some PRs from @ntdevlabs

**Comment by Vanqy9316** [comment]
@Karl-WE thank you for your help though. my [solution](https://github.com/ntdevlabs/tiny11builder/issues/310#issuecomment-2568435531) however is still valid and works. dont ask me why it works, but it does.

**Comment by ItzELECTR0** [comment]
>     * type `powershell -f` ,paste from clipboard and hit enter. the full command should look like this `powershell -f C:\Users\Vanqy\Desktop\Tools\tiny11builder-main\tiny11maker.ps1`

This should be mentioned in the README. While following the current README instructions you will get the Product Key issue and bash your head against the wall trying to find out why. (Took me a good while to find this solution)
